src/Parser/Paladin/Retribution/Modules/Items/WhisperOfTheNathrezim.js

```diff
@@ -78,7 +78,7 @@ class WhisperOfTheNathrezim extends Analyzer {
     when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
       return suggest(<Wrapper>Your usage of <ItemLink id={ITEMS.WHISPER_OF_THE_NATHREZIM.id} icon/> can be improved. Make sure to save up five holy power before your next <SpellLink id={SPELLS.JUDGMENT_CAST.id} icon/> window to get more time on the buff.</Wrapper>)
         .icon(ITEMS.WHISPER_OF_THE_NATHREZIM.icon)
-        .actual(`${formatPercentage(actual)}% damage contributed`)
+        .actual(<Wrapper>{formatPercentage(actual)}% of spenders with <SpellLink id={SPELLS.WHISPER_OF_THE_NATHREZIM_BUFF.id} icon/></Wrapper>)
```

Comment:
What's with the random Ret change? Was this intended?

Reply:
I changed the suggestion from % damage contributed to % of spenders with the buff from the legendary a while back but forgot to switch the text